articles/custom_storage_location.md

```diff
@@ -127,6 +127,41 @@ In **Permissions**, click **CORS configuration**. In the CORS configuration edit
 <br/>
 For more information, please read: [How Do I Configure CORS on My Bucket?](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors){:target="_blank"}
 
+### Setup bucket with AWS cloudformation
+
+You can also use AWS Cloudformation to provision a custom Synapse bucket.
+
+Instructions:
+1. Download the CF [template](https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/SynapseExternalBucket-v2.yaml).
+2. Use the [awscli](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/index.html) or
+[aws console](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create) to execute the template which will provision the bucket.
+
+Example using the awscli:
+```
+aws cloudformation create-stack \
+--stack-name MyCustomSynapseBucket \
+--template-body file://SynapseExternalBucket.yaml \
+--parameters ParameterKey=Department,ParameterValue=Cancer ParameterKey=Project,ParameterValue=Mammography \
+ParameterKey=OwnerEmail,ParameterValue=joe.smith@company.com ParameterKey=AllowWriteBucket,ParameterValue=true
```

Comment:
why is `ParameterKey=AllowWriteBucket,ParameterValue=true` required to fill out? wouldn't that be necessary in any case?

Reply:
This is to setup the bucket permission.  
AllowWriteBucket=true  means read-write bucket
AllowWriteBucket=false means read only bucket